thirdparty/libsimplewebm/libwebm/mkvparser/mkvparser.cc

```diff
@@ -1284,7 +1284,9 @@ long Segment::DoLoadCluster(long long& pos, long& len) {
     pos += cluster_size;
 
     m_pos = pos;
-
+    // -- GODOT start --
+    delete pCluster;
```

Comment:
Won't `pCluster` leak on line 1297 too if we don't enter this `if`?

Reply:
Nevermind, it's assigned to `m_pUnknownSize` and should not be freed.

Reply:
Yeah, I saw it too and dont decide to delete